### PR TITLE
Add note on timing issue in v5.x and earlier

### PR DESCRIPTION
**Pull request by DaveCTurner**

### CODE REVIEW

File: docs/resiliency/index.asciidoc [hunk 1]
```diff
@@ -170,6 +170,17 @@ shard.
 
 == Completed
 
+[float]
+=== Replicas can fail to replicate delete-and-reindex operations during certain network partitions (STATUS: DONE, V6.0.0)
```
Comment (bleskes) [inline docs/resiliency/index.asciidoc]: delete-and-reindex sounds too much related to the reindex API, maybe come up with a better name?

File: docs/resiliency/index.asciidoc [hunk 2]
```diff
@@ -170,6 +170,17 @@ shard.
...
+
+If a document is deleted and another document with the same id is indexed a
+short time late then it is possible for a replica to fail to replicate both of
```
Comment (bleskes) [inline docs/resiliency/index.asciidoc]: late*r*

Comment (bleskes) [reply docs/resiliency/index.asciidoc]: replica to fail to replicate sounds clanky

**Review by bleskes — CHANGES_REQUESTED** [review]
thanks for picking it up. I left some minor comments on the current text. My biggest concern is that it's a bit too vague - I think we should be more specific about what's going on (the replica will fail to process the index operation).

File: docs/resiliency/index.asciidoc (hunk 1 above)
Comment (jasontedor) [reply docs/resiliency/index.asciidoc]: I also think it makes it sound like a single operation when it is not.

**Comment by DaveCTurner** [comment]
Thanks, both. I added more words.

File: docs/resiliency/index.asciidoc [hunk 3]
```diff
@@ -170,6 +170,36 @@ shard.
...
+assigned version number 1 to the newly-indexed document, but the replica had
+not discarded its copy of the same record, then it would ignore the request
+from the primary to index the document under the mistaken impression that its
+version of the document was already up-to-date.
+
+In a healthy network this is extraordinarily unlikely to occur, but a
+badly-timed network partition of a particular duration may have delayed the
+replication of the deletion operation, causing this to happen with
+non-negligible probability. The introduction of sequence numbers
+{GIT}10708[#10708] means that this timing issue is no longer present.
```
Comment (jasontedor) [inline docs/resiliency/index.asciidoc]: I think that too strong of a statement is being made here.

Comment (DaveCTurner) [reply docs/resiliency/index.asciidoc]: In the sense that you think this issue is still present in 6.x?

Comment (jasontedor) [reply docs/resiliency/index.asciidoc]: Yes, I do.

**Pull request closed without merging**